service/application/src/main/java/com/solace/maas/ep/common/model/EventBrokerResourceConfiguration.java

```diff
@@ -1,14 +1,21 @@
 package com.solace.maas.ep.common.model;
 
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
 import lombok.Data;
 
 import java.util.List;
 
 @Data
 public class EventBrokerResourceConfiguration extends ResourceConfiguration {
+    @NotBlank
     private String id;
-    private String brokerType;
+    @NotBlank
     private String name;
-    private List<EventBrokerConnectionConfiguration> connections;
+    @NotEmpty
+    private List<@Valid EventBrokerConnectionConfiguration> connections;
+    @NotNull
     private ResourceConfigurationType resourceConfigurationType;
```

Comment:
All the model classes are updated to use Constraint Validation. This is much cleaner and IMO provides clear understanding of the validation requirements